features/name.feature

```diff
@@ -0,0 +1,10 @@
+Feature: As a User
+   I need to know that im on the name page
+
+   Background:
+      When I visit the name site
```

Comment:
This step is not needed and actually may conflict with the first step in the `Scenario`. You don't have to have a `Background` if it's not needed.